docs/pages/careers.tsx

```diff
@@ -158,6 +158,12 @@ const openRolesData = [
           'You will drive the development and maintenance of the documentation platform that powers all MUI products.',
         url: '/careers/react-engineer-docs-infra/',
       },
+      {
+        title: 'React Tech Lead — Core',
+        description:
+          'You will lead the development of MUI Core, positioning the library as the industry standard for design teams while doubling its adoption.',
```

Comment:
```suggestion
          'You will lead the development of the Core products, positioning them as the industry standard for product teams while doubling their adoption.',
```

- I'd avoid "MUI Core" and rather say "Core products" (I wonder if it should even be lowercase so it's not a thing—"core products" as in "main products").
- Talking in plural: "them" instead of "library"; "their" instead of "its" adoption
- "Design teams" → "Product teams" — this way, I think we encompass both designers and developers! "Design teams" can be confusing given this is not purely for designers